index.html

```diff
@@ -1,26 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
 </head>
+
 <body>
+
+    <!-- mobile hamburger menu -->
+    <div id="menu-list" class="mobile-menu-list">
+        <div id="menu-close-btn" class="menu-close-btn"><a>&#x2715</a></div>
+        <ul>
+            <li><a>Portfolio</a></li>
+            <li><a>About</a></li>
+            <li><a>Contact</a></li>
```

Comment:
- [ ] Please be careful here. You created the `a` tags for your list elements, but you have to include an `href` property to them in order to make them behave like a real link. So you should have something like `<a href="yourpath">`.
- [ ] Also, it would be nice to add some events listeners using JavaScript to these links, so every time you click a link, you have to make sure that you menu is closed again. So please apply the necessary changes.

Reply:
thank you,
all request changes done